ConsumerApi.Sdk/Endpoints/Relationships/RelationshipsEndpoint.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Backbone.ConsumerApi.Sdk.Endpoints.Common;
+using Backbone.ConsumerApi.Sdk.Endpoints.Common.Types;
+using Backbone.ConsumerApi.Sdk.Endpoints.Relationships.Types;
+
+namespace Backbone.ConsumerApi.Sdk.Endpoints.Relationships;
+
+public class RelationshipsEndpoint(EndpointClient client) : Endpoint(client)
+{
+    public async Task<ConsumerApiResponse<Relationship>> GetRelationship(string id) => await _client.Get<Relationship>($"Relationships/{id}");
+
+    public async Task<ConsumerApiResponse<List<Relationship>>> ListRelationships(PaginationFilter? pagination = null) => await _client.Get<List<Relationship>>("Relationships", null, pagination);
+    public async Task<ConsumerApiResponse<List<Relationship>>> ListRelationships(List<string> ids, PaginationFilter? pagination = null) => await _client
+        .Request<List<Relationship>>(HttpMethod.Get, "Relationships")
+        .Authenticate()
+        .WithPagination(pagination)
+        .AddQueryParameter("ids", ids)
+        .Execute();
+
+    public async Task<ConsumerApiResponse<RelationshipMetadata>> CreateRelationship(CreateRelationshipRequest request) => await _client.Post<RelationshipMetadata>("Relationships", request);
+
+    public async Task<ConsumerApiResponse<List<RelationshipChange>>> ListChanges(PaginationFilter? pagination = null) => await _client.Get<List<RelationshipChange>>("Relationships/Changes", null, pagination);
+
+    public async Task<ConsumerApiResponse<List<RelationshipChange>>> ListChanges(ListRelationshipChangesParameters parameters)
```

Comment:
Hmm, I get why you use a parameter object here. The list of parameters would be quite lengthy otherwise. BUT: I think I would still add all parameters separately - for the sake of consistency.

Reply:
👍